project_profitability_fields/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    "name": "Project Profitability Fields",
+    "summary": """
+        Add custom fields to track project profitability."
+    """,
+    "author": "Solvos",
+    "license": "AGPL-3",

```

Comment:
Better LGPL-3 in this case, because addon only depends on project (Odoo SA), that is LGPL-3 too. Remember our license choice criteria:
- When our addon depends on OCA one, that are usually AGPL-3, we must select AGPL-3 too.
- If there's no AGPL-3 licensed addons involved, we should choose LGPL-3, because is less restrictive

https://github.com/solvosci/slv-docker-odoo-prod/wiki/Odoo-SLV-contribution-brief-guidelines#24-manifest